### ISSUE TITLE
isFamilyFriendly 

### ISSUE THREAD

**Issue by BeatGrowth** [schemaorg/suggestions-questions-brainstorming#269]
What exactly defines Family Friendly? Can't find the definition. 

I'm assuming it means things link strong language, blood/gore, nudity, etc.. but i don't know for certain.

**Comment by danbri** [comment]
Yeah, it is culturally varying. Roughly it is suitable for children, but
that is notoriously impossible to define. If sites have a field
approximating this, it gives applications a chance to use it with
discretion, but we won’t try to impose anything globally via the schemas

**Comment by danbri** [comment]
I do not see an obvious action here, so I will convert this issue to a discussion.

If the Wikipedia entry for Family Friendly were better, we could link there.

One way to think about this is that Schema.org represents the common (ambiguous) practice of web pages declaring certain things to be "Family Friendly" but does not attempt to do what all else have also failed to do, i.e. define it in a clear and compelling manner that is universally agreed.

These discussions go back in W3C circles to PICS and https://www.w3.org/TR/NOTE-PICS-Statement and in general, have been around a long time.
